### PR TITLE
Fixes admin URLs for subclassed TextPlugins

### PR DESCRIPTION
**Pull request by jakob-o**
Changed code as to your suggestion @czpython regarding issue #413

### CODE REVIEW

**Comment by jakob-o** [comment]
Hey,

I adjusted the tests but obviously not correctly. Looks like the test run into 404s but I have no clue why... Would you mind pointing me in the right direction?

Regards

Jakob

File: djangocms_text_ckeditor/tests/test_plugin.py [hunk 1]
```diff
@@ -18,10 +18,16 @@
...
 from .base import BaseTestCase
+from djangocms_text_ckeditor.cms_plugins import TextPlugin
 
 
 class PluginActionsTestCase(BaseTestCase):
 
+    def get_custom_admin_url(self, plugin_class, name):
+        plugin_type = plugin_class.__name__.lower()
+        url_name = "%s_%s_%s" % (plugin_class.model._meta.app_label, plugin_type, name)
+        return url_name
```
Comment (czpython) [inline djangocms_text_ckeditor/tests/test_plugin.py]: missing a call to reverse() with admin namespace

Comment (jakob-o) [inline djangocms_text_ckeditor/tests/test_plugin.py]: Doh!

**Comment by czpython** [comment]
@jakob-o 
Just need to change the import as shown here https://travis-ci.org/divio/djangocms-text-ckeditor/jobs/197365066#L221

Don't worry about the other failure.

**Comment by jakob-o** [comment]
Done

**Pull request merged**